internal/connector/test/integration/features/connector-admin-api.feature

```diff
@@ -0,0 +1,50 @@
+Feature: connector admin api
+  In order to support the service
+  As an Admin API user
+  I need to be able to manage the service
+
+  Background:
+    Given the path prefix is "/api/connector_mgmt"
+
+    # org admin user used in admin API
+    Given an admin user named "Ricky Bobby" with roles "connector-fleet-manager-admin-full"
+
+    # regular user without roles
+    Given a user named "Evil Bob"
+
+  Scenario: Ricky lists connector types
+    Given I am logged in as "Ricky Bobby"
+
+    When I GET path "/v1/admin/kafka_connector_types"
+    Then the response code should be 200
+    And the ".kind" selection from the response should match "ConnectorTypeAdminViewList"
+    And the ".page" selection from the response should match "1"
+    And the ".size" selection from the response should match "2"
+    And the ".total" selection from the response should match "2"
+    And the ".items | length" selection from the response should match "2"
+    And the ".items[] | select(.id == "log_sink_0.1") | has("schema")" selection from the response should match "true"
+    And the ".items[] | select(.id == "log_sink_0.1") | .channels.stable.shard_metadata.connector_image" selection from the response should match "quay.io/mcs_dev/log-sink:0.0.1"
+    And the ".items[] | select(.id == "aws-sqs-source-v1alpha1") | has("schema")" selection from the response should match "true"
+    And the ".items[] | select(.id == "aws-sqs-source-v1alpha1") | .channels.stable.shard_metadata.connector_image" selection from the response should match "quay.io/mock-image:77c0b8763729a9167ddfa19266d83a3512b7aa8124ca53e381d5d05f7d197a24"
```

Comment:
The checks here do not test the full payload but validates that some of the constituents are part of the response.

Reply:
ltgm